static/templates/user_profile_modal.hbs

```diff
@@ -55,26 +66,41 @@
                         </div>
                         <div class="bottom">
                             <div id="content">
-                                {{#each profile_data}}
-                                    <div data-type="{{this.type}}" class="field-section custom_user_field" data-field-id="{{this.id}}">
-                                        <div class="name">{{this.name}}</div>
-                                        {{#if this.is_user_field}}
-                                            <div class="pill-container not-editable" data-field-id="{{this.id}}">
-                                                <div class="input" contenteditable="false" style="display: none;"></div>
-                                            </div>
-                                        {{else if this.is_link}}
-                                            <a href="{{this.value}}" target="_blank" rel="noopener noreferrer" class="value">{{this.value}}</a>
-                                        {{else if this.is_external_account}}
-                                            <a href="{{this.link}}" target="_blank" rel="noopener noreferrer" class="value">{{this.value}}</a>
-                                        {{else}}
-                                            {{#if this.rendered_value}}
-                                            <div class="value rendered_markdown">{{rendered_markdown this.rendered_value}}</div>
+                                {{#if is_bot}}
+                                    <div class="field-section">
+                                        <div class="name">{{#tr}}Bot type{{/tr}}</div>
+                                        <div class="bot_info_value" style="margin: 3px 0px;">{{bot_type}}</div>
```

Comment:
It's better to have this CSS in a CSS file.

Reply:
That was not needed CSS, fixed it.